apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -307,8 +322,9 @@ type DebugConfig struct {
 	// DebugLogLevel defines the log level which Contour will
 	// use when outputting log information.
 	//
-	// Contour's default is "info".
-	// +kubebuilder:validation:Enum=info;debug
+	// Values: `info` (default), `debug`.
+	//
+	// Other values will produce an error.
 	// +optional
 	DebugLogLevel LogLevel `json:"logLevel,omitempty"`
```

Comment:
Any opposition to renaming this field to just `LogLevel`? The YAML doesn't change.

Reply:
No opposition from me

Reply:
:+1: guess I'll do it in a separate PR